src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -178,7 +178,12 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
     {
         public Args args;
 
-        public SerializableReferenceType!(Notifier) notifier;
+        ubyte[Notifier.sizeof] notifier;
```

Comment:
Before, setting up a struct instance looked like:
```D
Request.UserSpecifiedParams params;
...
params.notifier.set(user_notifier); // mutates params
```
If `params` is declared const, it's not possible to call `params.notifier.set(...);`.